storey/dtypes.py

```diff
@@ -190,29 +191,111 @@ def __init__(self, delay_in_seconds=0, emission_type=EmissionType.All):
         self.delay_in_seconds = delay_in_seconds
         EmitBase.__init__(self, emission_type)
 
+    @staticmethod
+    def name():
+        return 'afterPeriod'
+
 
 class EmitAfterWindow(EmitBase):
     def __init__(self, delay_in_seconds=0, emission_type=EmissionType.All):
         self.delay_in_seconds = delay_in_seconds
         EmitBase.__init__(self, emission_type)
 
+    @staticmethod
+    def name():
+        return 'afterWindow'
+
 
 class EmitAfterMaxEvent(EmitBase):
     def __init__(self, max_events, emission_type=EmissionType.All):
         self.max_events = max_events
         EmitBase.__init__(self, emission_type)
 
+    @staticmethod
+    def name():
+        return 'maxEvents'
+
 
 class EmitAfterDelay(EmitBase):
     def __init__(self, delay_in_seconds, emission_type=EmissionType.All):
         self.delay_in_seconds = delay_in_seconds
         EmitBase.__init__(self, emission_type)
 
+    @staticmethod
+    def name():
+        return 'afterDelay'
+
 
 class EmitEveryEvent(EmitBase):
+    @staticmethod
+    def name():
+        return 'everyEvent'
+
     pass
 
 
+def _dict_to_emit_policy(policy_dict):
```

Comment:
Maybe it should fail on unexpected parameters. Overwise, someone might pass `'dealy': 10` and it will just be ignored.